src/main/java/mate/academy/service/UserService.java

```diff
@@ -2,19 +2,28 @@
 
 import mate.academy.model.User;
 
-public class UserService {
+public class UserService extends User {
```

Comment:
That's pretty cool you've managed to achieve some sort of inheritance here. However, I would say this is not the case here. We use inheritance when we have 'is-a' relationship. Here, it would be incorrect to say that our `UserService` is `User`.
You can read more [here](https://en.wikipedia.org/wiki/Inheritance_(object-oriented_programming))